src/applications/ask-va/containers/ResponseInboxPage.jsx

```diff
@@ -40,14 +40,40 @@ const ResponseInboxPage = ({ loggedIn }) => {
   const [sendReply, setSendReply] = useState({ reply: '', attachments: [] });
   const [loading, isLoading] = useState(true);
 
+  const options = {
+    body: JSON.stringify(sendReply),
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  };
+
+  const postApiData = url => {
+    isLoading(true);
+    return apiRequest(url, options)
+      .then(() => {
+        isLoading(false);
+      })
+      .catch(() => {
+        isLoading(false);
+        hasError(true);
+      });
+  };
+
   const handlers = {
     onInput: event => {
       setSendReply({ ...sendReply, reply: event.target.value });
     },
 
     onSubmit: () => {
       if (sendReply.reply) {
-        // TODO: Add endpoint to submit the reply
+        // Using a temporary test id provided by BE, will be replaced once inquiry endpoint is complete
```

Comment:
Per BE, we'll use this to test in staging for now until replies endpoint is built out